.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I don't think this file should be added.

Reply:
I've removed the .vscode file accidentally added.